node/coinstacks/solana/api/src/models.ts

```diff
@@ -3,7 +3,274 @@ import { BaseAccount, BaseTx, BaseTxHistory } from '../../../common/api/src' //
 /**
  * Contains info about a transaction
  */
-export interface Tx extends BaseTx {}
+export type Tx = RawTx & BaseTx
+
+export interface RawTx {
```

Comment:
helius-sdk exports `EnrichedTransaction` that we can can use in place of `RawTx` and all of the copied types.

Reply:
Wow nice! Changed in [774cde9](https://github.com/shapeshift/unchained/pull/1052/commits/774cde9191e9bca28050cd3c10aac6bd998b7721)